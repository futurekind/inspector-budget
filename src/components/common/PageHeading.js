import React from 'react';
import styled from 'styled-components';
import { colors } from '../../utils/styles'

const Heading = styled.h1`
    margin: 0;
    padding: 0;
    padding-bottom: 12px;
    color: ${colors.dark}
    font-weight: 300;
    font-size: 44px;
`

const PageHeading = ({
    children
}) => {
    return (
        <Heading>{ children }</Heading>
    )
}

export default PageHeading