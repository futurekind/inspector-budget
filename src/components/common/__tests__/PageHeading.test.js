import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import PageHeading from '../PageHeading';

describe('PageHeading', () => {

    it('renders correctly', () => {
        const wrapper = shallow(<PageHeading />)
        expect(toJSON(wrapper)).toMatchSnapshot()
    })

    it('renders its children', () => {
        const wrapper = shallow(<PageHeading>Some Heading</PageHeading>)
        expect(toJSON(wrapper)).toMatchSnapshot()
    })

})