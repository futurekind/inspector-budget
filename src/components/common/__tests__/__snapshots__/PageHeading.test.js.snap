exports[`PageHeading renders correctly 1`] = `<styled.h1 />`;

exports[`PageHeading renders its children 1`] = `
<styled.h1>
  Some Heading
</styled.h1>
`;
